lib/gui/lvgl/Kconfig.graphical

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2018-2019 Jan Van Winkel <jan.van_winkel@dxplore.eu>
+# Copyright (c) 2020 Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
+
+menu "Graphical settings"
+
+config LVGL_DISPLAY_DEV_NAME
+	string "Display device name"
+	default "DISPLAY"
+	help
+	  Name of the display device to use for rendering.
+
+config LVGL_HOR_RES
```

Comment:
Should we not also align `LVGL_HOR_RES` and `LVGL_VER_RES ` with LVGL defines  `LV_HOR_RES_MAX` and `LV_VER_RES_MAX`?

Reply:
@vanwinkeljan There are some more cases like this one (e.g. `LV_INDEV_DEF_READ_PERIOD` uses `CONFIG_LVGL_INPUT_REFRESH_PERIOD`). Should we do a complete alignment then, including stuff like `CONFIG_LVGL_USE_LED` for `LV_USE_LED` instead of `CONFIG_LVGL_OBJ_LED`? Initial idea was to just align object names.

Reply:
I would line up all of them instead of having a mix

Reply:
@vanwinkeljan I have adjusted all names now (I hope I haven't missed any). Can you re-review now? I'll split files once name changes are approved.

Reply:
@gmarull could it be you missed `Kconfig.graphical`?

Reply:
Added missing suffix _MAX for LVGL_HOR_RES and LVGL_VER_RES